### ISSUE TITLE
Regression in v0.13.8 for SVG with text node and no embedded font (take two)

### ISSUE THREAD

**Issue by akien-mga** [thorvg/thorvg#2706]
We found more SVG files which stopped working in ThorVG v0.13.8 because of the added text node support in 7e43cf402a4c5f585e96164661310a83f8d6eac4, and this problem is reproducible in v0.14.8.

Previous issue was #2468 and was fixed, but apparently this didn't solve the regression fully.

Godot bug report:
- https://github.com/godotengine/godot/issues/96491

Here's a set of SVG files which do not render anything with ThorVG 0.14.8 (since 7e43cf402 in 0.13.8), both in Godot and in the ThorVG Viewer (provided by @antimundo, from their MIT licensed open source game [poder-solar](https://github.com/antimundo/poder-solar)):

[thorvg-text-node-breaks-images.zip](https://github.com/user-attachments/files/16849750/thorvg-text-node-breaks-images.zip)

Here's one example from that set:

![power_plant](https://github.com/user-attachments/assets/3cc0d5d7-c17c-46a5-96a2-e57c09459414)

ThorVG v0.14.8 doesn't render it, while ThorVG v0.13.7 rendered it fine.

It does contain a text node, though I don't think it intends to display any text. This might have been added automatically by a plugin or copied over by mistake in a template SVG.

```xml
<text
       xml:space="preserve"
       style="font-size:37.3333px;font-family:Georgia;-inkscape-font-specification:Georgia;fill:#856944;fill-opacity:1;stroke:none;stroke-width:3;stroke-linejoin:round;stroke-dasharray:none;stroke-opacity:1;paint-order:stroke fill markers"
       x="982.45825"
       y="503.88559"
       id="text5"><tspan
         sodipodi:role="line"
         id="tspan5"
         x="982.45825"
         y="503.88559"
         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-family:Impact;-inkscape-font-specification:Impact" /></text><text
       xml:space="preserve"
       style="font-size:18.6667px;font-family:'Franklin Gothic Medium';-inkscape-font-specification:'Franklin Gothic Medium, ';text-align:center;letter-spacing:0px;text-anchor:middle;opacity:1;fill:#399351;fill-opacity:1;stroke:none;stroke-width:4;stroke-linecap:round;stroke-linejoin:miter;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;paint-order:stroke fill markers"
       x="596.26978"
       y="1831.0868"
       id="text81"><tspan
         sodipodi:role="line"
         id="tspan81"
         x="596.26978"
         y="1831.0868" /></text>
```

Removing that node fixes the problem, but like in #2468, our expectation is that an incomplete/invalid text node shouldn't prevent rendering the rest of the valid SVG nodes.

Other SVG renderers don't seem to have problem with it (browsers, file managers, inkscape, etc.).

**Comment by akien-mga** [comment]
CC @mgrudzinska

**Comment by capnm** [comment]
@mgrudzinska There is an issue with the transformations inside a group of other SVG elements, even after loading an external font.

![Screenshot from 2024-09-03](https://github.com/user-attachments/assets/0961fdfe-15a2-4dc6-8da2-59fa89909644)

![icon_looks_left_bad](https://github.com/user-attachments/assets/0a524066-43a4-4cdb-aeff-8d283f129442)

**Comment by hermet** [comment]
Confirmed that all addressed resources were fixed in v0.14.9.

**Comment by akien-mga** [comment]
Thanks @mgrudzinska!
